src/main/java/com/exactpro/th2/hand/services/WinActionsBuilder.java

```diff
@@ -368,6 +370,34 @@ public static void addWinTableClick(CSVPrinter printer, RhWinActionsMessages.Win
 		printer.printRecord(values);
 	}
 
+	public static void addWinTableSearch(CSVPrinter printer, RhWinActionsMessages.WinTableSearch winTableSearch) throws IOException
+	{
+		List<String> headers = new ArrayList<>(), values = new ArrayList<>();
+
+		headers.add(ACTION);
+		values.add("TableSearch");
+
+		addDefaults(winTableSearch.getId(), winTableSearch.getExecute(), headers, values);
+		addLocator(winTableSearch.getLocatorsList(), headers, values);
+
+		headers.add("#filter");
+		values.add(winTableSearch.getSearchParams());
+
+		headers.add("#column");
+		values.add(winTableSearch.getTargetColumn());
+
+		headers.add("#firstrowindex");
+		values.add(winTableSearch.getFirstRowIndex());
```

Comment:
getFirstRowIndex is primitive type and it cannot be null. So maybe if it has default value should we exclude it?